app/src/main/java/com/eduard/caloriecounter/presentation/presenter/CollectionContract.java

```diff
@@ -5,20 +5,16 @@
 public interface CollectionContract {
 
      interface View extends BasePresenter.View {
-
          void initView();
          void setViewData(String data);
-
     }
 
     interface Presenter {
-
-        void onClick(android.view.View view);
-
+         void onClick(android.view.View view);
```

Comment:
Non informative method name. AND VERY BAD IDEA TO USE onClick AS NAME OF METHOD!!!!